commit d8ca20ea66ef1c30cfe4b707b73363873035362f
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2019-03-22T18:31:43+00:00 (7 months ago)
    
    MANTA-4172 zk client ping timeouts not working
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/package.json b/package.json
index 53c1c51..365e500 100644
--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     },
     "dependencies": {
         "manta": "1.2.6",
-        "manatee": "git+https://github.com/joyent/manatee.git#35be0bca5871d9ba35f68c75b0ec1e8aa8538f42",
+        "manatee": "git+https://github.com/joyent/manatee.git#33b33433070e1c83622ca6a555b4bc38f1eec0e6",
         "carrier": "0.2.0"
     },
     "sdcDependencies": {
